src/main/resources/db.changelog/changelog.xml

```diff
@@ -0,0 +1,188 @@
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="context" type="varchar(1000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="creation_time" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="url_photo" type="datetime">
+            </column>
+        </createTable>
+
+        <sql>CREATE TYPE friendship_state AS ENUM ('pending', 'accepted', 'rejected')</sql>
+
+        <createTable tableName="friendship">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="user1_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user2_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="friendship_state">
+                <constraints nullable="false"/>
+            </column>
+            <column name="request_date" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="acceptance_date" type="datetime"/>
+        </createTable>
+
+        <createTable tableName="comments">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="post_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="comment" type="varchar(1000)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="likes">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="likes_comments">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints primaryKey="true" />
+            </column>
+            <column name="comment_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="likes_posts">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints primaryKey="true" />
+            </column>
+            <column name="post_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
```

Comment:
I would prefer like to comment and like to post to be represented as 'MANY TO MANY' relationship